<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeSenatorSwitchWidget</class>
 <widget class="QWidget" name="ChangeSenatorSwitchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>39</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>senatoradd</string>
   </property>
  </widget>
  <widget class="QComboBox" name="candidate2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>45</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="senator2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>45</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="candidate3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="senator1">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="candidate1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>10</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
   </property>
   <property name="text">
    <string>更换为</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>45</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
   </property>
   <property name="text">
    <string>更换为</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>106</x>
     <y>80</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: rgb(137,129,161);
font: 11px &quot;Microsoft YaHei UI Light&quot;;}</string>
   </property>
   <property name="text">
    <string>更换为</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="senator3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>80</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>40</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="BtnStyle" stdset="0">
    <string>senatordel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
